core/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -1190,6 +1190,51 @@ class QualificationSuite extends BaseTestSuite {
     }
   }
 
+  test("custom reasons for operators disabled by default") {
+    TrampolineUtil.withTempDir { outParquetFile =>
+      TrampolineUtil.withTempDir { eventLogDir =>
+        val (eventLog, _) = ToolTestUtils.generateEventLog(eventLogDir, "unsup") { spark =>
+          import spark.implicits._
+          val data = Seq((1, ("Person1", 30)), (2, ("Person2", 25))).toDF("id", "person")
+          data.write.parquet(s"$outParquetFile/person_info")
+          val df = spark.read.parquet(s"$outParquetFile/person_info")
+          val dfToJson = df.withColumn("person_json", to_json($"person"))
+          dfToJson
+        }
+
+        TrampolineUtil.withTempDir { outpath =>
+          val appArgs = new QualificationArgs(Array(
+            "--output-directory",
+            outpath.getAbsolutePath,
+            eventLog))
+
+          val (exit, _) =
+            QualificationMain.mainInternal(appArgs)
```

Comment:
could we put `val (exit, _) = QualificationMain.mainInternal(appArgs)` in one line?